scripts/get_command_info.py

```diff
@@ -0,0 +1,210 @@
+# Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+import argparse
+import json
+import os
+from os.path import join
+
+"""Valkey command categorizer
+
+This script analyzes command info json files and categorizes the commands based on their routing. The output can be used
+to map commands in the cluster_routing.rs#base_routing function to their RouteBy category. Commands that cannot be 
+categorized by the script will be listed under the "Uncategorized" section. These commands will need to be manually
+categorized.
+
+To use the script:
+1. Clone https://github.com/valkey-io/valkey
```

Comment:
maybe script should checkout the repo automatically

Reply:
I think its good enough like that, we will deal with it when we integrate it into the build process